fil_fungible_token/src/token/types.rs

```diff
@@ -18,172 +18,204 @@ pub enum ActorError<Err> {
 pub type Result<T, E> = std::result::Result<T, ActorError<E>>;
 
 /// A standard fungible token interface allowing for on-chain transactions that implements the
-/// FRC-XXX standard. This represents the external interface exposed to other on-chain actors
+/// FRC-0046 standard. This represents the external interface exposed to other on-chain actors
 ///
-/// Token authors should implement this trait and link the methods to standard dispatch numbers in
-/// their actor's `invoke` entrypoint. A standard helper (TODO) is provided to aid method dispatch.
-///
-/// TODO: make non-pseudo code
-/// ```
-/// //struct Token {}
-/// //impl FrcXXXToken for Token {
-/// //    ...
-/// //}
-///
-/// //fn invoke(params: u32) -> u32 {
-/// //    let token = Token {};
-///
-/// //    match sdk::message::method_number() {
-/// //        1 => constructor(),
-/// //        2 => token.name(),
-/// //        _ => abort!()
-/// //        // etc.
-/// //    }
-/// //}
-/// ```
-pub trait FrcXXXToken<E> {
+/// Token authors must implement this trait and link the methods to standard dispatch numbers (as
+/// defined by [FRC-0042](https://github.com/filecoin-project/FIPs/blob/master/FRCs/frc-0042.md)).
+pub trait Frc46Token<E> {
```

Comment:
I don't think Rust has opinions about acronyms, like Go does. But I do! Could we capitalise the FRC acronym please?